src/cider/exec/nextgen/jitlib/llvmjit/LLVMJITEngine.h

```diff
@@ -31,6 +31,7 @@ struct CompilationOptions {
   bool optimize_ir = true;
   bool aggressive_jit_compile = true;
   bool dump_ir = false;
+  bool enable_vectorize = false;
```

Comment:
Q: why default false?

Reply:
I think it will increase codegen cost significantly, only enable it when necessary.